go/libraries/utils/async/action_executor.go

```diff
@@ -70,6 +70,13 @@ type work struct {
 	wg  *sync.WaitGroup
 }
 
+// ClearError removes the cached error if one is present.
+func (aq *ActionExecutor) ClearError() {
```

Comment:
Unused

Reply:
I added it as I wanted a clean way to remove errors when someone used the modified `WaitForEmpty`, but I guess I don't need it anymore.